ee/dtr/admin/configure/external-storage/nfs.md

```diff
@@ -36,23 +39,23 @@ docker run -it --rm {{ page.dtr_org }}/{{ page.dtr_repo }}:{{ page.dtr_version }
   <other options>
 ```
 
-Use the format `nfs://<nfs server>/<directory>` for the NFS storage URL. To support **NFS v4**, you can now specify additional options when running [docker/dtr install](../../../../../reference/dtr/2.6/cli/install/) with `--nfs-storage-url`.
+Use the format `nfs://<nfs server>/<directory>` for the NFS storage URL. To support **NFS v4**, you can now specify additional options when running [docker/dtr install](/reference/dtr/2.6/cli/install/) with `--nfs-storage-url`.
 
 When joining replicas to a DTR cluster, the replicas will pick up your storage
 configuration, so you will not need to specify it again.
 
 ### Reconfigure DTR to use NFS
 
-When upgrading from a previous version of DTR that is already using
-NFS, you can continue using the same configurations. To support **NFS v4**, additional NFS reconfiguration options have been added to the CLI. See [docker/dtr reconfigure](../../../../../reference/dtr/2.6/cli/reconfigure/) for more details.
+To support **NFS v4**, more NFS options have been added to the CLI. See [New Features for 2.6.0 - CLI](/ee/dtr/release-notes/#260) for updates to [docker/dtr reconfigure](/reference/dtr/2.6/cli/reconfigure/). 
 
-
-To take advantage of the new DTR built-in support for NFS, you can
-reconfigure DTR to use NFS:
+There is currently an issue with [reconfiguring and restoring DTR with `--nfs-storage-url`](/ee/dtr/release-notes#version-26) which leads to erased tags. Make sure to [back up your DTR metadata](/ee/dtr/admin/disaster-recovery/create-a-backup/#back-up-dtr-metadata) before you proceed. To work around the issue, manually create a storage volume. If DTR is already installed in your cluster, [reconfigure DTR](/reference/dtr/2.6/cli/reconfigure/) with the `--dtr-storage-volume` flag using your newly-created volume. If you're [restoring DTR](/reference/dtr/2.6/cli/restore/) from an existing backup, use `docker/dtr restore` with `--dtr-storage-volume` and the new volume. See [How to recover a lost NFS blob storage mount with DTR 2.6](https://success.docker.com/article/how-to-recover-a-lost-nfs-blob-storage-mount-with-dtr-26) for more details.  
```

Comment:
Please note affected version, as `currently` will become dated.

Make this a warning?

Reply:
KB URL may change.  Please keep an eye on the KB draft.